src/examples/LargeSnapshot/LargeSnapshot.csproj

```diff
@@ -2,9 +2,10 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Why was the downgrade necessary?

Reply:
because the CI/CD is still using .net 6.0 SDK

Reply:
its easier to downgrade the project target framework than to upgrade the CI/CD script, that might create some side effect

Reply:
ok, but we should move this project to .NET 8 at some point